src/App.css

```diff
@@ -30,34 +30,3 @@
     transform: rotate(360deg);
   }
 }
-
-/* Calvin's CSS */
-
-a,
-a:hover,
-a:active,
-a:focus,
-a:visited {
-  color: #222;
-  text-decoration: none;
-  cursor: pointer;
-}
-
-h3 {
-  margin: 0;
-}
-
-label {
```

Comment:
can you leave this `label` CSS? otherwise the inline form elements are just going to make me mad :)